pkg/runctl/run/interfaces.go

```diff
@@ -21,7 +21,8 @@ type Manager interface {
 
 // Run represents a pipeline run
 type Run interface {
-	GetStartTime() *metav1.Time
+	IsDeleted() bool
+	GetJFRStepStartTime() *metav1.Time
```

Comment:
Interface should not make assumptions about single steps in the run.

Reply:
I reverted the name change. I also moved the documentation to the interfaces.